app/models/Like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_like_counter
+
+  def update_like_counter
+    post.update(likes_counter: post.likes.size)
+  end
```

Comment:
## Optional
- [ ] This method should be private after adding [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save`